app/views/planning_applications/_decision_notice.html.erb

```diff
@@ -16,8 +16,8 @@
         {
           "Applicant" => planning_application.applicant_name,
           "Application number" => planning_application.reference,
-          "Application received" => planning_application.received_at,
-          "Decision date" => planning_application.determination_date || "TBD",
+          "Application received" => planning_application.received_at.to_fs,
+          "Decision date" => planning_application.determination_date.to_fs || "TBD",
```

Comment:
I think if the determination date is nil this won't behave as intended — the `||` works for the previous code being nil, but here the `to_fs` will blow up instead. Safe navigation should do the trick?

```suggestion
          "Decision date" => planning_application.determination_date&.to_fs || "TBD",
```